source/import-css/index.html.md.erb

```diff
@@ -32,9 +32,11 @@ Then link the CSS file inside the `<head>` tag of your HTML page or page templat
 To import all the Sass rules from GOV.UK Frontend, add the following to your Sass file:
 
 ```scss
-@import "node_modules/govuk-frontend/dist/govuk/all";
+@import "node_modules/govuk-frontend/dist/govuk/index";
 ```
 
+You do not need `/index` at the end of your import if you’re using Dart Sass, LibSass 3.6.0 or higher, or Ruby Sass 3.6.0 or higher.
```

Comment:
🙌🏻  Makes sense to be consistent with the rest of the site about this. I'll create an [issue to remove statements](https://github.com/alphagov/govuk-frontend-docs/issues/491) about LibSass 3.6.0 and Ruby Sass 3.6.0 cross site.

Reply:
I wasn't sure about this one. We want to encourage folks to not use deprecated packages but also give folks the correct info if they are and aren't in a position to move off them. I feel like these docs lean more towards the latter.